packages/hoprd/src/api/v2.ts

```diff
@@ -38,10 +39,16 @@ export async function setupRestApi(
 
   // assign internal objects to each requests so they can be accessed within
   // handlers
-  service.use(urlPath, (req, _res, next) => {
-    req.context = new Context(node, stateOps)
-    next()
-  })
+  service.use(
+    urlPath,
+    function addNodeContext(req, _res, next) {
+      req.context = { node: this.node, stateOps: this.stateOps }
+      next()
+    }
+      // Need to explicitly bind the instances to the function
+      // to make sure the right instances are present
+      .bind({ node, stateOps })
```

Comment:
this wasn't an issue before, why is it now?

Reply:
We changed the transpilation target. So the output of Typescript has massively changed. Hence some language constructs behave slightly differently. Especially regarding scope and extent.